src/main/scala/beam/agentsim/infrastructure/ParkingStall.scala

```diff
@@ -27,6 +27,7 @@ object ParkingStall {
 
   val CostOfEmergencyStallInDollars: Double = 50.0
 
+  @SuppressWarnings(Array("UnusedMethodParameter")) // LOOKS LIKE SCAPEGOAT BUG
```

Comment:
Did you log it with them perchance?

Reply:
There was some "confusing" original code. After change as following the error disappear
![image](https://user-images.githubusercontent.com/12264098/132122801-6a8112e1-3dce-4450-b3f0-ba754e82c968.png)